application-services/custom/camera-management/README.md

```diff
@@ -22,27 +120,11 @@ make install-edge-video-analytics
 # Run the EVAM services (in another terminal)
 make run-edge-video-analytics
 # ...
-# Leave this running. If needed to stop
+# OPTIONAL: Leave this running. If needed to stop
```

Comment:
```suggestion
# OPTIONAL: Leave this running. Stop if needed.
```

Reply:
I dont think that wording works either. Maybe something like `If you need to stop the services, run:`

Reply:
Hmm, actually it looks like Ctrl-C will stop it, and that command removes the stopped containers.